lib/zip-codes.rb

```diff
@@ -1,18 +1,30 @@
 require 'yaml'
 
 module ZipCodes
-  VERSION = '0.2.1'
+  VERSION = '0.2.2'
```

Comment:
This may not be a breaking change (semver-1), but _if not_ it's certainly structural enough to warrant a semver-2 change. 

```suggestion
  VERSION = '0.3.0'
```

Reply:
IMO this is enough of a major change that server-1 is appropriate.  But that's just my interpretation